packages/dhis2/src/Utils.js

```diff
@@ -1,4 +1,13 @@
 import { composeNextState } from '@openfn/language-common';
+import _ from 'lodash';
+
+export const { indexOf, isObject, isArray } = _;
```

Comment:
Hey this is kinda weird- why are we re-exporting some lodash functions from this file?

It's cool to use lodash but why not just import the functions we need, where we need them? :thinking: 

Also, these are very low-value imports. What's wrong with `Array.isArray()` or  `[].indexOf` ?

Reply:
@josephjclark 
My thinking was since lodash already have `isArray()` function we can use that instead of creating our own `isArray()` function. 

So i moved lodash to `Util.js` and export the functions that are needed for both `Util.js` and `Adaptor.js`

Reply:
But Javascript has an isArray function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Works great and doesn't require any complicated exports